stock_inventory_quantity_history/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Don't you usually add Akretion copyright here ? Moreover, you can set 'https' instead.

Reply:
I've added it, thanks